### DISCUSSION TITLE
using just few skeleton unstyled components

### DISCUSSION THREAD

**Discussion by FOSSgeek** [saadeghi/daisyui#1404] [Q&A] [answered]
i want to use only few skeleton unstyled  components and keep the rest styled for example i want to use styled button but not styled navbar, thank you for the awesome library.

**Comment by saadeghi** [comment, marked as answer]
Currently there's only one config for that and it's for all the components.  
So if you only need some of the components as styled, there are 2 ways to do that:
- having 2 separate Tailwind CSS builds, one with daisyUI `styled` config on and the other one off. And you can also add prefixes to one of them so the class names wouldn't mix.
- Manually copy/paste the the CSS from daisyUI source: [components](https://github.com/saadeghi/daisyui/tree/master/src/components) and [utilities](https://github.com/saadeghi/daisyui/tree/master/src/utilities)

Let me know if you have any questions

**Reply by FOSSgeek** [reply]
saad you are my legend and your made me saed ( haapy in arabic), thank you so much i wish some day i get enough money to donate for daisy ui
